packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -1634,6 +1634,21 @@ export default class Meeting extends StatelessWebexPlugin {
     return this.#isoLocalClientMeetingJoinTime;
   }
 
+  /**
+   * Setter - sets isoLocalClientMeetingJoinTime
+   * This will be set once on meeting join, and not updated again
+   * @param {string | undefined} isoLocalClientMeetingJoinTime
+   */
+  set isoLocalClientMeetingJoinTime(isoLocalClientMeetingJoinTime: string | undefined) {
+    if (!this.#isoLocalClientMeetingJoinTime) {
```

Comment:
question (just to double check): in a scenario where user joins, leaves and then joins the same meeting again, are we supposed to still use the fist meeting join time and not the 2nd one?

Reply:
You are right in calling this out, this is a problem. I made the mistake of not recalling that the meeting object is not always destroyed on leave (in the case of OBTP or meeting ongoing) and was only considering the guest case. I will need to make adjustments for this. Ill let you know once done, I'm still waiting for locus to make their change.